scripts/bitbucket.py

```diff
@@ -104,7 +104,16 @@ def post_result(url, user, password, verify, data, debug):
             err("SSL warnings disabled\n")
 
     # Construct the URL and JSON objects
-    post_url = url + commit_sha
+    if driver == 'Bitbucket Server':
+        post_url = j['source']['bitbucket_url'] + 'rest/build-status/1.0/commits/' + commit_sha
+    elif driver == 'Bitbucket Cloud':
+        owner = j['source']['owner']
```

Comment:
The `owner` variable doesn't appear to be in the README. It would make sense to default `owner` to be the same value as `bitbucket_username` if no value is provided.